src/language-html/parser-html.js

```diff
@@ -105,6 +107,21 @@ function ngHtmlParser(
         }
       }
     }
+  } else if (isVueHtml) {
+    // If not Vue SFC, treat as html
+    recognizeSelfClosing = true;
+    normalizeTagName = true;
+    normalizeAttributeName = true;
+    allowHtmComponentClosingTags = true;
+    isTagNameCaseSensitive = false;
+    const htmlParseResult = parser.parse(input, {
+      canSelfClose: recognizeSelfClosing,
+      allowHtmComponentClosingTags,
+      isTagNameCaseSensitive,
+    });
```

Comment:
HTML parser `normalizeAttributeName: true`, should we apply?

Reply:
`normalizeAttributeName` is not parser option. Is is used for AST transformation after parse.

Reply:
Okay, I don't know that.